src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results.html

```diff
@@ -97,6 +97,50 @@
               </p>
             </div>
 
+            <div ng-if="record._source._extra_data.record_matches.length > 1 
+            && record._source._workflow.data_type === 'hep' " ng-init="showPossibleMatches = true">
+                <div ng-show="showPossibleMatches">
+                    <div ng-if="record._source._extra_data.record_matches.length <= 5">
```

Comment:
The matched identifiers will be from now in ``record._source._extra_data.matches.fuzzy``